src/redux/items/items.js

```diff
@@ -8,25 +9,21 @@ export const itemSlice = createSlice({
   },
   reducers: {
     addItem: (state, action) => {
-      return { ...state, items: [action.payload, ...state.items] }
+      state.items.push({ id: nanoid(11), ...action.payload })
     },
     deleteItem: (state, action) => {
-      const items = state.items.filter(item => item.name !== action.payload.name)
-      return { ...state, items: [...items] }
+      state.items = state.items.filter(item => item.name !== action.payload.name)
     },
     updateStatus: (state, action) => {
       const index = state.items.findIndex(item => item.name === action.payload.name)
-      return {
-        ...state,
-        items: [
-          ...state.items.slice(0, index),
-          { ...state.items[index], status: 'Borrowed' },
-          ...state.items.slice(index + 1)
-        ]
-      }
+      state.items[index].status = 'Borrowed'
+    },
+    editItem: (state, action) => {
```

Comment:
1) there's no 'key' attribute in the items anymore, so i'm not sure this find will work
2) you're replacing the existing item in the store right? i think the reason you're getting a console error is you're not creating a new id for the new item. see `addItem`

Reply:
yup, fixed it! will push soon